### ISSUE TITLE
Suggestion: Publish it on addons.mozilla.org

### ISSUE THREAD

**Issue by sbrl** [go-shiori/shiori-web-ext#6]
The extension is great, but it would be really cool if it were to be published on addons.mozilla.org. Then people can receive updates automatically without having to re-click a link in the GitHub releases page every time it updates.

**Comment by RadhiFadlillah** [comment]
Yep, I'm planned to put it in both Firefox and Chrome. However, right now I'm a bit busy so expect it at most maybe in next month.

**Comment by sbrl** [comment]
Sure :-)

**Comment by cenk1cenk2** [comment]
Will this ever be possible, I can still use the extensions mostly without a problem, but if they were published it would be synced across the devices, which would be nice.

**Comment by l0b0** [comment]
It seems to be [published](https://addons.mozilla.org/en-GB/firefox/addon/shiori_ext/) now. Can this be closed?

**Comment by fmartingr** [comment]
I don't currently have plans to publish it myself until the add-on reaches some maturity, and I would need to do so while not having all the keys to the castle myself, in case I need to give access to someone else. I have no experience publishing things to browser stores, so this may take a while. 

Anything that would ease out the experience of installing the addon for you folks in the time being?

**Comment by mirkoperillo** [comment]
@fmartingr this issue is not valid anymore, it can be closed

**Comment by fmartingr** [comment]
Thank you again, @mirkoperillo
